client/src/components/HostViews/Recap.vue

```diff
@@ -49,6 +52,14 @@ export default {
     winningResponse() {
       const LAST_INX = this.winningResponses.length - 1
```

Comment:
Last_inx? Can you either make it more clear what this is shortening or write a comment?

Reply:
I will just use LAST_INDEX